### ISSUE TITLE
can not download weights

### ISSUE THREAD

**Issue by fatalfeel** [KupynOrest/DeblurGAN#203]
Download weights from Google Drive . Note that during the inference you need to keep only Generator weights.
Put the weights into

**Comment by chihina** [comment]
I also have the same issue. Please anyone share a Generator model weight file!

**Comment by fatalfeel** [comment]
I will modify code let it simple and train it by myself, later paste here
DeblurGAN have v2 but its not original gan

**Comment by Carrolinewang** [comment]
> I also have the same issue. Please anyone share a Generator model weight file!

Have you solve the problem? I have the same issue.

**Comment by fatalfeel** [comment]
i find out in DeblurGAN/checkpoints/experiment_name/
latest_net_D.pth
latest_net_G.pth 
so you do not need to download, the latest_net_G.pth  is the pretrain

**Comment by Carrolinewang** [comment]
> i find out in DeblurGAN/checkpoints/experiment_name/
> latest_net_D.pth
> latest_net_G.pth
> so you do not need to download, the latest_net_G.pth is the pretrain

I get it.Thank you.

**Comment by fatalfeel** [comment]
here i modify Perceptual-Loss that refer from 
1. https://github.com/tyui592/Perceptual_loss_for_real_time_style_transfer/blob/master/main.py
2. https://github.com/rrmina/fast-neural-style-pytorch/blob/master/train.py
change struct make DeblurGAN easy learn and train
https://github.com/fatalfeel/DeblurGAN

later~ I will release FPN-Resnet to do G-net, its faster on cuda and do small block well

**Comment by fatalfeel** [comment]
we do not need to move to  DeblurGANV2, 
I add hi-resolution fpn152 with resnet-backbone here is the code. 
https://github.com/fatalfeel/DeblurGAN/blob/master/models/fpn_resnet.py
Also add n_layers_G layers up options and n_blocks_G up options
